<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>541</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Rubik Cube Scanner</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>0</y>
      <width>291</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="12" column="0">
      <widget class="QPushButton" name="nextScannedFace">
       <property name="text">
        <string>New Face</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QPushButton" name="saveScannedFace">
       <property name="text">
        <string>Save/Replace face </string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Scanned face:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>45</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="9" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="0">
      <widget class="QListWidget" name="facesListView"/>
     </item>
     <item row="6" column="0">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>32</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0" alignment="Qt::AlignHCenter">
      <widget class="CVImageWidget" name="facesView" native="true">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>130</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>130</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QPushButton" name="finishScan">
       <property name="text">
        <string>Finish Scan</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Scanned faces:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="CVImageWidget" name="OpenCVViewer" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>640</width>
      <height>480</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1280</width>
      <height>720</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>968</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCalibration">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionCalibrate_color_detection"/>
    <addaction name="actionChange_colors"/>
   </widget>
   <addaction name="menuCalibration"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCalibrate_color_detection">
   <property name="text">
    <string>Calibrate color detection</string>
   </property>
  </action>
  <action name="actionChange_colors">
   <property name="text">
    <string>Change colors</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CVImageWidget</class>
   <extends>QWidget</extends>
   <header>cvimagewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
